src/elements/content-sidebar/messages.js

```diff
@@ -72,6 +72,16 @@ const messages = defineMessages({
         defaultMessage: 'Modify General Task',
         description: 'modal title for when editing an existing general task',
     },
+    securityBlockedErrorMessage: {
+        defaultMessage: 'This action is unavailable due to a security policy.',
+        description: 'Tooltip text for when Box Sign is blocked due to a security policy',
+        id: 'be.contentSidebar.securityBlockedError',
+    },
+    watermarkBlockedErrorMessage: {
```

Comment:
Can we rename these from `ErrorMessage` to `Tooltip` to further clarify their purpose?